ElectronClient/app/gui/ConfigScreen.jsx

```diff
@@ -413,6 +413,24 @@ class ConfigScreenComponent extends React.Component {
 					{descriptionComp}
 				</div>
 			);
+		} else if (md.type === Setting.TYPE_BUTTON) {
+			const theme = themeStyle(this.props.theme);
+			const buttonStyle = Object.assign({}, theme.buttonStyle, {
+				display: 'inline-block',
+				marginRight: 10,
+			});
+
+			return (
+				<div key={key} style={rowStyle}>
+					<div style={labelStyle}>
+						<label>{md.label()}</label>
+					</div>
+					<button style={buttonStyle} onClick={md.onClick}>
+						Edit
```

Comment:
This should be a translatable string.

Reply:
Oh you are right. Besides calling the `_(...)` function, how do I go about adding that translation?

Reply:
You don't have to do anything other than wrapping it in the `_(` function. The string `Edit` should already have been translated and added to the .pot file. After the PR, we have to update the translations anyway, because you added new strings as well.